src/robot/libraries/Collections.py

```diff
@@ -783,7 +783,7 @@ def dictionary_should_not_contain_value(self, dictionary, value, msg=None,
         )
 
     def dictionaries_should_be_equal(self, dict1, dict2, msg=None, values=True,
-                                     ignore_keys=None, ignore_case=False):
+                                     ignore_keys=None, ignore_case=False, ignore_value_order=False):
```

Comment:
This row seems to be over our maximum 88 character recommendation. Please change it to something like below and go through also other long lines.
```python
ignore_keys=None, ignore_case=False,
ignore_value_order=False):
```

Reply:
resolved